testing/single/src/showCallHierarchy.py

```diff
@@ -48,8 +48,10 @@ def method(self):
 class BaseConsumer:
     def consumer(self, base: Base):
         base.method()
+        return base
```

Comment:
Why is this added?

Reply:
see `#test basic call hierarchy ...` cell.